src/de/jost_net/JVerein/io/ZipMailer.java

```diff
@@ -240,8 +240,8 @@ public void run(ProgressMonitor monitor) throws ApplicationException
                       
                 Mail ml = (Mail) Einstellungen.getDBService()
                         .createObject(Mail.class, null);
-                ml.setBetreff(betreff);
-                ml.setTxt(txt);
+                ml.setBetreff(wtext1.getBuffer().toString());
+                ml.setTxt(wtext2.getBuffer().toString());
```

Comment:
```suggestion
                ml.setBetreff(wtext1.toString());
                ml.setTxt(wtext2.toString());
```
Man kann einfach toString() machen. Der Zugriff auf den StringBuffer ist nicht notwendig.

Reply:
war schon so,, darum hatte ich es so übernommen. Hab es jetzt geändert, auch oben.